services/ui-src/package.json

```diff
@@ -20,7 +20,7 @@
     "@emotion/styled": "^11.11.5",
     "@hookform/resolvers": "^2.9.11",
     "@vitejs/plugin-react": "^4.3.0",
-    "aws-amplify": "^5.3.19",
+    "aws-amplify": "^5.3.4",
```

Comment:
This is still downgrading the minimum version, we probably want to keep the min version at 5.3.19 unless we have a reason to downgrade

Reply:
changed this to go back to 5.3.19